test/filtering.test.js

```diff
@@ -252,6 +262,34 @@ describe('array data provider', () => {
     expect(getBodyCellContent(grid, 0, 0).innerText).to.equal('bar');
   });
 
+  // TODO: find better option to check rather than grid._cache.items...
```

Comment:
In case grid is attached, we could expose and use `getVisibleItems(grid).length` from `helpers`.

Reply:
Done